### PR TITLE
Added a visualize function in utils.py

### PR DESCRIPTION
**Pull request by xinwei89**
This tries to address #32

### CODE REVIEW

File: aerialseg/utils.py [hunk 1]
```diff
@@ -226,3 +228,31 @@ def assemble_coco_json(
         ]
 
     return coco_json
+
+
+def visualize_or_save_image(image: str, predictor, meta, png_out: str = ""):
```
Comment (mauch) [inline aerialseg/utils.py:233]: Just wondering if you should make `meta` and optional argument here? That way the caller doesn't have to create a `MetadataCatalog` object before calling this function even if they don;t care about plotting the class names.

**Review by sahandv — APPROVED** [review]
Thanks @xinwei89 .
Everything works.

**Review by mauch — APPROVED** [review]
Cool. I'm happy too - just a minor comment about requiring a `MetadataCatalog` to be passed to the function which you are welcome to ignore.

File: aerialseg/utils.py (hunk 1 above)
Comment (xinwei89) [reply aerialseg/utils.py]: Thanks, I have made the default `None`

**Pull request merged**